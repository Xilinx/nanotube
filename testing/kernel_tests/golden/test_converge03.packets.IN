###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
2020-03-26 17:44:17.184404
0000  7f 02 04 06 08
2020-03-26 17:45:34.076120
0000  70 0f 04 06 08
2020-03-26 17:45:47.762601
0000  00 02 04 06 08
