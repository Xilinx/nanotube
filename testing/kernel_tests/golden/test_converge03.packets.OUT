###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
1970-01-01 00:00:00.000000
0000  7f 02 04 06 05
1970-01-01 00:00:00.000000
0000  70 0f 04 06 06
1970-01-01 00:00:00.000000
0000  00 02 04 06 00
1970-01-01 00:00:00.000000
0000  70 0f 04 06 06
1970-01-01 00:00:00.000000
0000  00 02 04 06 00
